### DISCUSSION TITLE
How to add Separator in ContextMenu when Binding

### DISCUSSION THREAD

**Discussion by cornerbowlsoftware** [AvaloniaUI/Avalonia#16868] [General]
I have a collection of menu item view models which work great.  When I add a null value to my collection, I want a Separator to display instead of my custom template.  In WPF this is handled with a style trigger.  How can I do this in Avalonia?  

Style:
```
	<Style Selector="MenuItem" x:DataType="lib:MenuItemViewModel">
		<Setter Property="Header" Value="{Binding Text}"/>
		<Setter Property="Command" Value="{Binding Command}"/>
		<Setter Property="InputGesture" Value="{Binding InputGestureText, Converter={StaticResource StringToKeyGestureConverter}}"/>
		<Setter Property="Icon" Value="{Binding PathIcon, Converter={StaticResource PathToIconConverter}}"/>
	</Style>
```

**Comment by cornerbowlsoftware** [comment]
I was able to finally get the separator to display by setting the bound Text value to "-" as documented.

**Reply by tqk2811** [reply]
Have you found a solution yet?
